apps/site/components/EOL/VulnerabilitiesTable/index.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames';
+import { useTranslations } from 'next-intl';
+import type { FC } from 'react';
+
+import VulnerabilityChip from '#site/components/EOL/VulnerabilityChips/Chip';
+import LinkWithArrow from '#site/components/LinkWithArrow';
+import type { Vulnerability } from '#site/types/vulnerabilities';
+
+const VulnerabilitiesTable: FC<{
+  vulnerabilities: Array<Vulnerability>;
+  maxWidth?: string;
+}> = ({ vulnerabilities, maxWidth = 'max-w-2xs' }) => {
+  const t = useTranslations();
+
+  return (
+    <table className="w-full">
+      <thead>
+        <tr>
+          <th>{t('components.eolModal.table.cves')}</th>
+          <th>{t('components.eolModal.table.severity')}</th>
+          <th>{t('components.eolModal.table.overview')}</th>
+          <th>{t('components.eolModal.table.details')}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {vulnerabilities.map((vulnerability, i) => (
+          <tr key={i}>
+            <td>
+              {vulnerability.cve.map(cveId => (
+                <div key={cveId}>
+                  <LinkWithArrow
+                    href={`https://www.cve.org/CVERecord?id=${cveId}`}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    {cveId}
+                  </LinkWithArrow>
+                </div>
+              ))}
+
+              {vulnerability.cve.length > 0 || '-'}
+            </td>
+            <td>
+              <VulnerabilityChip severity={vulnerability.severity} />
+            </td>
+            <td className={classNames(maxWidth, 'truncate')}>
+              {vulnerability.description || vulnerability.overview || '-'}
+            </td>
+            <td>
+              {vulnerability.ref ? (
```

Comment:
This somehwat makes me think that this is a React ref, what is `ref`? reference? Can we rename this property to something more meaningful?

Reply:
Might be asking much but can we map this on our generator to another name? I can just imagine this bugfging someone's else head as it did mine.